main/src/com/google/refine/commands/recon/ReconJudgeOneCellCommand.java

```diff
@@ -190,10 +190,19 @@ protected HistoryEntry createHistoryEntry(long historyEntryID) throws Exception
             newCell.recon.judgmentBatchSize = 1;
 
             if (judgment == Judgment.None) {
-                newCell.recon.judgment = Recon.Judgment.None;
+                if (cell.recon.error == null) {
+                    newCell.recon.judgment = Recon.Judgment.None;
+                } else {
+                    newCell.recon.judgment = Recon.Judgment.Error;
+                }
                 newCell.recon.match = null;
-
                 description = "Discard recon judgment for " + cellDescription;
+
+            } else if (judgment == Judgment.Error) {
+                newCell.recon.judgment = Recon.Judgment.Error;
+                newCell.recon.match = null;
+
+                description = "Errors in recon judgment for " + cellDescription;
```

Comment:
This case would correspond to the user manually setting the recon object to an error state, but I don't think that makes sense. Errors are meant to be generated by the reconciliation process itself, not added manually by users.
So I would say we should throw an exception in this case (such as `InvalidArgumentException`).